lib/screens/settings/settings_screen.dart

```diff
@@ -74,16 +76,15 @@ class SettingsScreen extends StatelessWidget {
             ),
           ],
         );
-      }
+      },
     );
   }
 
-  Future<void> _showChangePasswordDialog(BuildContext context) async {
+  Future<void> _showChangePasswordDialog(BuildContext theContext) async {
```

Comment:
If we want to name `BuildContext`s like this, let's use `topContext` instead of `theContext`.

Reply:
sure. will change asap

Reply:
Done